bootstrap-scripts/raspberry-pi-setup.sh

```diff
@@ -62,4 +62,6 @@ echo Install node bindings for cmake
 sudo npm install -g cmake-js
 
 # Configure abigail-device service
-sudo cp $mydir/config-files/abigail-device.service /lib/systemd/system
```

Comment:
I haven't checked but I think it didn't work because it was trying to get the file from `bootstrap-scripts/config-files`